system/include/emscripten/wasmfs.h

```diff
@@ -32,6 +35,10 @@ int wasmfs_create_file(const char* pathname __attribute__((nonnull)), mode_t mod
 // Returns 0 on success like `mkdir`, or a negative value on error.
 int wasmfs_create_directory(const char* path __attribute__((nonnull)), mode_t mode, backend_t backend);
 
+// Unmounts the directory at a specific path.
+// Returns 0 on success, or a negative value on error.
```

Comment:
It would be good to mention in the comment that the directory must be a mount point.